app/src/main/res/layout/episodes_list_fragment.xml

```diff
@@ -47,14 +47,15 @@
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
-    <ProgressBar
+    <com.google.android.material.progressindicator.LinearProgressIndicator
```

Comment:
Are you sure that this one should be linear?

Reply:
No, that seems like an oversight on my end